addons/sale_pdf_quote_builder/models/ir_actions_report.py

```diff
@@ -138,8 +138,8 @@ def _get_form_fields_mapping(self, order):
         }
 
         # Adding fields from each line, prefixed by the line_id to avoid conflicts
-        for line in order.order_line:
-            form_fields_mapping.update(self._get_sol_form_fields_mapping(line))
```

Comment:
Still need to update the mapping... >.<

Reply:
Shoot, thanks for your patience- gotta get better at catching stuff like this before sending it back for review

Reply:
Never hesitate to test it, even in local =]